variable/replace.go

```diff
@@ -0,0 +1,34 @@
+package variable
+
+import (
+	"log"
+	"regexp"
+	"strings"
+)
+
+// varRegex holds the compiled regular expression for matching variables
+var varRegex *regexp.Regexp
+
+func init() {
+	varRegex = regexp.MustCompile(VariableRegexp)
+}
```

Comment:
You don't technically need the init function here: https://github.com/tarmac-project/tarmac/blob/main/pkg/app/server.go#L16

Reply:
Interesting! I never knew it could be used like this, thanks.